recipes-wpe/wpeframework/include/wpeframework-ocdm-playready.inc

```diff
@@ -0,0 +1,17 @@
+SUMMARY = "WPE Framework OpenCDMi module for playready"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=66fe57b27abb01505f399ce4405cfea0"
+
+require wpeframework-plugins.inc
+
+DEPENDS += " playready"
```

Comment:
Could you please use DEPENDS_append here?

Reply:
done